content/howto/bash.md

```diff
@@ -0,0 +1,969 @@
+Title: HOWTO Bash
```

Comment:
ogólnie jest ogrom wiedzy tutaj. Bardzo super. Napisane jest to w przystępny sposób. Porozdzielałabym by każdy rozdział by był czymś osobnym. Jeżeli ktoś zobaczy ogromny plik to może zrezygnować z rozpoczęcia czytania go. A gdy widzi ze jest mniejszy to może go zachęci :)